test/integration/scheduler/util.go

```diff
@@ -77,7 +77,7 @@ func initDisruptionController(t *testing.T, testCtx *testutils.TestContext) *dis
 
 	informers.Start(testCtx.Scheduler.StopEverything)
 	informers.WaitForCacheSync(testCtx.Scheduler.StopEverything)
-	go dc.Run(testCtx.Scheduler.StopEverything)
+	go dc.Run(testCtx.Ctx)
```

Comment:
I'd like to revert this for now. Meanwhile, I can take a look at the scheduler code to see if we can eliminate `Scheduler.StopEverything` entirely, and then revisit this file. WDYT?

Reply:
The compilation would fail without the change. What is the timeframe you're looking at for making these scheduler changes?

Reply:
I noticed that the original context obj is initialized in testCtx in [InitTestAPIServer](https://github.com/kubernetes/kubernetes/blob/a6f695581b36ec2349cf7d3e8e65f2e5288a30c9/test/integration/util/util.go#L330), and afterwards get passed to [scheduler.New()](https://github.com/kubernetes/kubernetes/blob/a6f695581b36ec2349cf7d3e8e65f2e5288a30c9/test/integration/util/util.go#L418). In other words, here textCtx.Ctx.Done() == testCtx.Scheduler.StopEverything. So no more questions, it's safe to use testCtx.Ctx here.